backend/plan/views.py

```diff
@@ -9,37 +9,11 @@
 import plan.examples as examples
 from courses.models import Section
 from courses.util import get_course_and_section, get_current_semester
-from courses.views import CourseList
-from PennCourses.docs_settings import PcxAutoSchema
-from plan.filters import CourseSearchFilterBackend
+from PennCourses.docs_settings import PcxAutoSchema, reverse_func
 from plan.models import Schedule
-from plan.search import TypedCourseSearchBackend
 from plan.serializers import ScheduleSerializer
 
 
-class CourseListSearch(CourseList):
-    """
-    The main course API endpoint for PCP. Without any GET parameters, it simply returns all courses
-    for a given semester. There are a few filter query parameters which constitute ranges of
-    floating-point numbers. The values for these are <min>-<max> , with minimum excluded.
-    For example, looking for classes in the range of 0-2.5 in difficulty, you would add the
-    parameter difficulty=0-2.5. If you are a backend developer, you can find these filters in
-    backend/plan/filters.py/CourseSearchFilterBackend. If you are reading the frontend docs,
-    these filters are listed below in the query parameters list (with description starting with
-    "Filter").
-    """
-
-    schema = PcxAutoSchema(
-        examples=examples.CourseListSearch_examples,
-        response_codes={
-            "/api/plan/courses/": {"GET": {200: "[SCHEMA]Courses listed successfully."}}
-        },
-    )
-
-    filter_backends = [TypedCourseSearchBackend, CourseSearchFilterBackend]
```

Comment:
This might be further down, but does this get replicated anywhere? The PCP search endpoint does actually add filters on top of the base CourseList, so want to make sure we still have that.

Reply:
Yeah it is replicated here:

https://github.com/pennlabs/penn-courses/pull/260/files/e9edc68b35b84a5d2e8cbf0d135e00d69fae3c77#diff-52fd16120332ef10fba01c425d68eff2f29194a6a128ac1c231fb0f856deb6c5R132

The new url is `/api/base/<slug:semester>/search/courses/`